charts/postgres-operator/crds/operatorconfigurations.yaml

```diff
@@ -472,6 +472,8 @@ spec:
                     type: string
                   logical_backup_s3_secret_access_key:
                     type: string
+                  logical_backup_s3_access_secret_name:
```

Comment:
maybe we change the name to `logical_backup_s3_secret_name`. Then it's in alphabetical order (which I prefer in CRD schema) and you don't have to move things around again.